zipkin-ui/templates/layout.mustache

```diff
@@ -1,42 +1,28 @@
-    <div id='navbar' class='navbar navbar-inverse' role='navigation'>
-      <div class='container'>
-        <div class='navbar-header'>
+  <nav class='navbar navbar-expand-lg navbar-dark bg-dark'>
           <a class='navbar-brand' href='{{contextRoot}}'>
-            Zipkin<span class='muted' style='font-size: .75em; padding-left: 10px;' data-i18n="nav.inves">Investigate system behavior</span>
+          <img src="https://zipkin.io/public/img/zipkin-logo-200x119.jpg" width="60" height="40" class="d-inline-block align-top rounded" alt=""/>
+            ZIPKIN <span class='font-weight-light navbar-text' style="font-size: .75em;" data-i18n="nav.inves">Investigate system behavior</span>
```

Comment:
Does this change the `Zipkin` in the upper left of the UI to uppercase?

Reply:
Changed it to logo only